src/server/index.js

```diff
@@ -2,14 +2,25 @@
 
 const express = require('express');
 const path = require('path');
+const mongodb = require('mongodb');
 
 const PORT = process.env.PORT || 8080;
 const app = express();
 
 app.use(express.static(path.resolve(__dirname, '../../dist')));
 
 app.get('/heartbeat', (req, res) => {
-  res.json({status: 'ok'});
+  var response;
+  var MongoClient = mongodb.MongoClient;
+  var url = 'mongodb://localhost:27017/epam'; 
```

Comment:
please use the .env config. Other users can use different ports and urls and also on the satging environment we will use different url